src/pages/projects.js

```diff
@@ -422,6 +422,41 @@ const ProjectDescriptions = () => (
 
     <TitleWithAnchor headerClassName="projects-title projects-common-title">Emerging Projects</TitleWithAnchor>
 
+    <div className="project-box">
+      <TitleWithAnchor headerClassName="project-major-title projects-common-title" headerTag="h3">hBPF</TitleWithAnchor>
+      <div className="project-minor-title">
+        hBPF = eBPF in hardware
```

Comment:
nit: I understand that this is part of the project tagline, but I wonder whether the `=` gives the wrong impression to folks visiting this page; vendors have been running eBPF in hardware for some time now without this project, so equating these two with the `=` seems definitive without recognising the existance of alternatives. Would you be open to some slighty modification here?

```suggestion
        hBPF - eBPF in hardware
```

Reply:
I'm fine with using '-' instead of '='.